lib/src/exception.dart

```diff
@@ -12,3 +12,11 @@ class RedisException implements Exception {
   @override
   String toString() => 'RedisException: $message';
 }
+
+/// Exception thrown if there was an attempt to send a command after the
+/// connection was closed.
+class ConnectionClosedException implements Exception {
```

Comment:
```suggestion
class RedisConnectionClosedException implements RedisException {
```
Starting the exception name with Redis and extending from the existing one could be better here